public/js/reducers/ui.js

```diff
@@ -29,7 +29,7 @@ const State = makeRecord(({
 
 function update(state = State(), action: Action) : Record<SidebarsState> {
   switch (action.type) {
-    case C.COLLAPSE_SIDEBAR: {
+    case C.TOGGLE_SIDEBAR: {
```

Comment:
@jlongster heh that's a habit from my work project we always refer to our constants as `C` and I noticed we called immutable `I` so it seemed convenient. 😆